app/controllers/plantings_controller.rb

```diff
@@ -124,16 +124,14 @@ def update_days_before_maturity(planting, crop_id)
   end
 
   def plantings
-    @plantings = if @owner
-                   @owner.plantings
-                 elsif @crop
-                   @crop.plantings
-                 else
-                   Planting
-                 end
-
-    @plantings = @plantings.current unless @show_all
-    @plantings = @plantings.includes(:owner, :crop, :garden).order(:created_at).paginate(page: params[:page])
-    @plantings
```

Comment:
none would rely on that behaviour, because i made this method only a few days ago in another PR.